src/components/VmsList/Vms.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react'
+import { connect } from 'react-redux'
+
+import style from './style.css'
+import Vm from './Vm'
+
+const Vms = ({ vms }) => {
+  const selectedVmId = vms.get('selected')
+  const containerClass = 'container-fluid container-cards-pf ' + (selectedVmId ? style['move-left'] : style['move-left-remove'])
```

Comment:
note to self: we should really look into using https://github.com/JedWatson/classnames for such computed styles